seL4-platforms/builds.py

```diff
@@ -451,8 +451,8 @@ def mq_run(success_str: str,
 
 
 def mq_lock(machine: str) -> List[str]:
-    """Get lock for a machine."""
-    return ['time', 'mq.sh', 'sem', '-wait', machine, '-k', job_key()]
+    """Get lock for a machine. Allow lock to be reclaimed after 30min."""
+    return ['time', 'mq.sh', 'sem', '-wait', machine, '-k', job_key(), '-T', '1200']
```

Comment:
Is this seconds, so 1800 then for 30 mins?

Reply:
Indeed, that doesn't match. I'll change it to 1800 seconds, because that's fewer places to change and this should only be necessary in very rare cases anyway. If jobs fail or get stuck in a "normal" way (i.e. no network outage) there are already the standard timeouts in place for that.